__tests__/_helper.ts

```diff
@@ -21,6 +19,18 @@ import { createJsonResponse } from '../src/utils'
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/serlo.org-cloudflare-worker for the canonical source repository
  */
+
+import { Json } from 'fp-ts/lib/Either'
```

Comment:
```suggestion
```

There are eslint errors in your workspace. Run `yarn lint` before you make a new push to the PR.